frontend/src/styles/Features.css

```diff
@@ -0,0 +1,17 @@
+body {
```

Comment:
Can you make these more detailed to this component. I think left as is, this css will affect all body,h1,and li html elements.